src/PerfView64/App.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Diagnostics;
+
+namespace PerfView64
+{
+    public class App
```

Comment:
Can you put a comment on the basic architecture here (it is a wrapper that is referring to the other EXE (and is meant to be deployed in the same directory as PerfView.exe.